tests/integrations/python/tests/test_anthropic.py

```
@@ -2810,6 +2811,67 @@ def test_49_web_search_sources_validation(self, anthropic_client, test_config, p
         
         print(f"✓ Sources validation test passed!")
 
+    # =========================================================================
+    # Async Inference Tests
+    # =========================================================================
+
+    @pytest.mark.parametrize(
+        "provider,model", get_cross_provider_params_for_scenario("simple_chat")
+    )
+    def test_50_async_messages(self, anthropic_client, test_config, provider, model):
```

Ruff (0.15.1)

[warning] Unused method argument: `test_config`

(ARG002)